turbot/table_turbot_mod_version.go

```diff
@@ -0,0 +1,140 @@
+package turbot
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableTurbotModVersion(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "turbot_mod_version",
+		Description: "Module versions in turbot organization.",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "name", Require: plugin.Optional},
+				{Name: "status", Require: plugin.Optional},
+				{Name: "filter", Require: plugin.Optional},
+				{Name: "org_name", Require: plugin.Optional},
+			},
+			Hydrate: listModVersion,
+		},
+		Columns: []*plugin.Column{
+			// Top columns
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "The name of the mod."},
+			{Name: "identity_name", Type: proto.ColumnType_STRING, Description: "The indentity name of the mod."},
+			{Name: "org_name", Type: proto.ColumnType_STRING, Transform: transform.FromQual("org_name"), Description: "The name of the organization."},
```

Comment:
What does `org_name` point to? Will it always be Turbot or is it specific to the organization?

Reply:
It will always not be Turbot it is specific to the organization.

Reply:
Thanks for the confirmation @Joeturbot